.github/workflows/scorecard.yml

```
@@ -0,0 +1,73 @@
+# This workflow uses actions that are not certified by GitHub. They are provided
+# by a third-party and are governed by separate terms of service, privacy
+# policy, and support documentation.
+
+name: Scorecard supply-chain security
+on:
+  # For Branch-Protection check. Only the default branch is supported. See
+  # https://github.com/ossf/scorecard/blob/main/docs/checks.md#branch-protection
+  branch_protection_rule:
+  # To guarantee Maintained check is occasionally updated. See
+  # https://github.com/ossf/scorecard/blob/main/docs/checks.md#maintained
+  schedule:
+    - cron: '24 3 * * 5'
```

```suggestion
    - cron: '25 3 * * 5'
```

GitHub Actions scheduled jobs have a maximum granularity of five minutes, so this would run at most 0320 hours or 0325 hours, won't it?

---

@agriyakhetarpal No, it just runs at 03:25 AM every Friday.